### PR TITLE
docs(*) [PDE-5460] Remove docs from monorepo

### PR DESCRIPTION
**Pull request by FokkeZB**
Prepare to move docs to Mintlify.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,10 +18,9 @@ It consists of a few main packages:
 ## Docs
 
 * Public-facing docs:
-  - [Latest CLI developer guide](packages/cli/README.md)
```
Comment (FokkeZB) [inline README.md:21]: Link to Mintlify for docs (both UI and CLI)

File: packages/cli/README.md
Comment (FokkeZB) [inline packages/cli/README.md:1]: Changed to something like `schema/README.md`, with links to docs.

File: packages/cli/scripts/docs.js
Comment (FokkeZB) [inline packages/cli/scripts/docs.js:1]: Changed to now only generate `docs/cli.md` and also no HTML versions anymore.

File: docs/index.html
Comment (FokkeZB) [inline docs/index.html:1]: Left but changed to redirect from https://zapier.github.io/zapier-platform-cli/ to Mintlify.

File: packages/cli/src/generators/templates/callback/creates/prediction.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ module.exports = {
 
           // This is NOT how you normally do authentication. This is just to demo how to write a create here.
           // Refer to this doc to set up authentication:
-          // https://zapier.github.io/zapier-platform-cli/#authentication
```
Comment (FokkeZB) [inline packages/cli/src/generators/templates/callback/creates/prediction.js:36]: Will update this once we know where this section lands.

File: packages/cli/src/tests/utils/convert.js [hunk 3]
```diff
@@ -47,11 +47,11 @@ const visualAppDefinition = {
           },
           {
             source:
-              "// Configure a request to an endpoint of your api that\n// returns custom field meta data for the authenticated\n// user.  Don't forget to congigure authentication!\n\nconst options = {\n  url: 'https://api.example.com/custom_field_meta_data',\n  method: 'GET',\n  headers: {\n    'Accept': 'application/json'\n  },\n  params: {\n\n  }\n}\n\nreturn z.request(options)\n  .then((response) => {\n    const results = response.data;\n\n    // modify your api response to return an array of Field objects\n    // see https://zapier.github.io/zapier-platform-schema/build/schema.html#fieldschema\n    // for schema definition.\n\n    return results;\n  });\n",
```
Comment (FokkeZB) [inline packages/cli/src/tests/utils/convert.js:50]: Will update this once we know where this section lands.

File: packages/core/README.md
Comment (FokkeZB) [inline packages/core/README.md:1]: A few updates for consistency (and drop broken Travis badge)

File: packages/legacy-scripting-runner/README.md
Comment (FokkeZB) [inline packages/legacy-scripting-runner/README.md:1]: A few updates for consistency (and drop broken Travis badge)

File: packages/schema/README.md
Comment (FokkeZB) [inline packages/schema/README.md:1]: A few updates for consistency (and drop broken Travis badge)

**Review by FokkeZB — COMMENTED** [review]
Self-review

**Review by eliangcs — APPROVED** [review]
🔥

**Pull request merged**